scout/server/config.py

```diff
@@ -68,7 +68,7 @@
 # FEATURE FLAGS
 SHOW_CAUSATIVES = True
 SHOW_OBSERVED_VARIANT_ARCHIVE = True
-SHOW_ALAMUT_LINK = True
+HIDE_ALAMUT_LINK = False
```

Comment:
If you don't have this param at all then you see the Alamut button. Good so we don't have to do anything. In a future PR (not this one) we should document the new params added to the config file!

Reply:
Good point. I added a few lines to the admin doc. We can revisit that page later, but now there will at least be something there.